services/response_code.proto

```diff
@@ -1529,4 +1529,9 @@ enum ResponseCodeEnum {
      * The node account is not allowed to be updated
      */
     UPDATE_NODE_ACCOUNT_NOT_ALLOWED = 359;
+
+    /*
+     * The token has no metadata or supply key
+     */
```

Comment:
@derektriley  Please make sure to merge services PR first. Then merge this but undelete the branch name. Then open a services PR with change in branch name to `main`